docs/providers/kubeless/guide/debugging.md

```diff
@@ -0,0 +1,164 @@
+
+```python
+import urllib2
+import json
+
+def find(request):
+    term = request.json["term"]
+    url = "https://feeds.capitalbikeshare.com/stations/stations.json"    
+    response = urllib2.urlopen(url)
+    stations = json.loads(response.read())
+    
+    hits = []
+    
+    for station in stations["stationBeanList"]:
+        if station["stAddress1"].find(term) > -1:
+            hits.append(station)
+
+    return json.dumps(hits)
+```
+
+And its corresponding Serverless YAML file:
+
+```yml
+# serverless.yml
+service: bikesearch
+provider:
+  name: kubeless
+  runtime: python2.7
+
+plugins:
+  - serverless-kubeless
+
+functions:
+  bikesearch:
+    handler: handler.find
+```
+
+Let's invoke correctly that function
+
+```
+serverless invoke --function bikesearch --data '{"term":"Albemarle"}' -l
+
+# Output
+Serverless: Calling function: bikesearch...
+--------------------------------------------------------------------
+[ { availableDocks: 6,
+    totalDocks: 15,
+    city: '',
+    altitude: '',
+    stAddress2: '',
+    longitude: -77.079382,
+    lastCommunicationTime: '2017-08-25 04:46:09 AM',
+    postalCode: '',
+    statusValue: 'In Service',
+    testStation: false,
+    stAddress1: 'Tenleytown / Wisconsin Ave & Albemarle St NW',
+    stationName: 'Tenleytown / Wisconsin Ave & Albemarle St NW',
+    landMark: '',
+    latitude: 38.947607,
+    statusKey: 1,
+    availableBikes: 9,
+    id: 80,
+    location: '' } ]
+```    
+
+What happens when something goes wrong? The function currently has no error handling, so that's easy enough to test. Let's invoke the function again with a typo (use *trm* as the name of the input parameter instead of *term*):
+
+```
+serverless invoke --function bikesearch --data '{"trm":"Albemarle"}' -l
+
+# Output
+Serverless: Calling function: bikesearch...
+ 
+  Serverless Error ---------------------------------------
+ 
+  Internal Server Error
+ 
+  Get Support --------------------------------------------
+     Docs:          docs.serverless.com
+     Bugs:          github.com/serverless/serverless/issues
+     Forums:        forum.serverless.com
+     Chat:          gitter.im/serverless/serverless
+ 
+  Your Environment Information -----------------------------
+     OS:                     darwin
+     Node Version:           8.3.0
+     Serverless Version:     1.20.2
+```
+
+Serverless returned an error message with a 500 server code, which is what you would expect from a web framework. However, it would be useful to see Python stack trace to better debug the source of the error. To achieve this, first find the container where the function is running using *kubectl*:
+
+```
+kubectl get pods
+
+# Output
+NAME                          READY     STATUS             RESTARTS   AGE
+bikesearch-3894699941-mbt2t   1/1       Running            1          2d
+```
+
+Once the name of the container is known, use *kubectl logs* to check the logs:
+
+```
+kubectl logs bikesearch-3894699941-mbt2t
```

Comment:
you don't need to obtain the pod and call kubectl directly, you can obtain the same executing `sls logs -f bikesearch`